File: app/src/main/java/com/example/assignment/views/RegisterActivity.java
package com.example.assignment.views;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.assignment.R;
import com.example.assignment.databinding.ActivityRegisterBinding;

public class RegisterActivity extends AppCompatActivity {

    private ActivityRegisterBinding activityRegisterBinding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        activityRegisterBinding = ActivityRegisterBinding.inflate(getLayoutInflater());
        setContentView(activityRegisterBinding.getRoot());
    }
}

File: app/src/main/java/com/example/assignment/views/RegisterActivity.kt
package com.example.assignment.views

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.assignment.databinding.ActivityRegisterBinding

class RegisterActivity : AppCompatActivity() {
    private var activityRegisterBinding: ActivityRegisterBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        activityRegisterBinding = ActivityRegisterBinding.inflate(
            layoutInflater
        )
        setContentView(activityRegisterBinding!!.root)
    }
}